test/integration/01-common.sh

```diff
@@ -30,15 +27,7 @@ identity_foreign=d8e2f047-99b7-48e8-89d1-0e9b6e0b2464
 # start testing
 #
 
-testing start "$PACKAGENAME"
-testing class "package"
```

Comment:
Current pipeline has no sudo rights, nor its building package. These things will be done in #131